app-full-microprofile/src/main/java/com/example/quarkus/secure/TestSecureController.java

```diff
@@ -49,7 +49,7 @@ private static String generateJWT(String key) {
         JWTAuth provider = JWTAuth.create(null, new JWTAuthOptions()
                 .addPubSecKey(new PubSecKeyOptions()
                         .setAlgorithm("RS256")
-                        .setSecretKey(key)));
+                        .setBuffer(key))); // Set up private key
```

Comment:
Method name feels weird, checked with https://vertx.io/docs/vertx-auth-jwt/java/ and that's the way.

Is there any migration document for jump from Vert.x 3.9 to Vert.x 4.1?
It should be at least linked in Release notes for RHBQ 2.x

Reply:
Migration guide mentioning a link to Vertx Migration Guide. Method is recommended from class source comments https://github.com/vert-x3/vertx-auth/blob/master/vertx-auth-common/src/main/java/io/vertx/ext/auth/PubSecKeyOptions.java